### ISSUE TITLE
Prefect! Works Fine, You Can Also Bypass The Antivirus By Forcing It To Run As A ADMIN But /webcam Dosent Work Fix it :)

### ISSUE THREAD

**Issue by SkidsLAMO** [Rdimo/DiscordRAT#20]

**Comment by Rdimo** [comment]
yes I'm aware that the webcam doesn't work, the reason is because I don't have a webcam myself so when I tested it out I had to connect my phone and use that as my webcam and that worked out fine but apparently it doesn't work for people with real webcams, so ill maybe start working on polishing this rat even more

**Comment by SkidsLAMO** [comment]
Alr
